src/dmd/tocvdebug.d

```diff
@@ -91,31 +91,24 @@ uint cv4_memfunctypidx(FuncDeclaration fd)
     //printf("cv4_memfunctypidx(fd = '%s')\n", fd.toChars());
 
     type *t = Type_toCtype(fd.type);
-    AggregateDeclaration ad = fd.isMember2();
-    if (ad)
+    if (AggregateDeclaration ad = fd.isMember2())
     {
         // It's a member function, which gets a special type record
 
-        idx_t thisidx;
-        if (fd.isStatic())
-            thisidx = dttab4[TYvoid];
-        else
-        {
-            assert(ad.handleType());
```

Comment:
Isn't this changing the behavior as now `null` could be passed to `Type_toCtype`?

https://github.com/dlang/dmd/blob/806f105da1d802919f5741ecc1568c9623e5377f/src/dmd/toctype.d#L253-L261

AFAICT there's nothing ensuring that the type of the `AggregateDeclaration` isn't null?

Reply:
It's a reasonable point. I added it back.